WordPress/Classes/Utility/BuildInformation/FeatureFlag.swift

```diff
@@ -2,6 +2,7 @@
 /// different builds.
 @objc
 enum FeatureFlag: Int, CaseIterable, OverrideableFlag {
+    case bloggingPrompts
```

Comment:
Yes, I intentionally put this at the top to avoid having to scroll to the bottom past all the old flags every time.